middleware/verify_jwt.js

```diff
@@ -0,0 +1,35 @@
+import jwt from 'jsonwebtoken';
+import { cartDao } from '../model';
+const TOKEN_KEY = '' + process.env.SECRET_KEY;
+
+const validateToken = async (req, res, next) => {
+  try {
+    let bearer, token;
```

Comment:
```suggestion
    let token;
```
언더바를 사용하면 더이상 `bearer`이라는 변수를 사용하지 않게 되므로 삭제하셔도 좋겠네요 :)

Reply:
감사합니다! 수정하겠습니다.